web/src/app/shared/models/pullRequest.model.ts

```diff
@@ -10,13 +10,14 @@ import { UserModel } from './user.model';
 export class PullRequestModel {
   uid: string = '';
   url: string = '';
-  state: 'open' | 'closed';
+  state: string;
```

Comment:
I think it is better to fix these states as there are a limited amount of options than to have it just as a `string`

Reply:
In that scenario i have to add 5 values here. I will add it

Reply:
ok great 👍